ecs-image-build/apispec/company-delta-spec.yml

```diff
@@ -42,6 +42,8 @@ components:
           type: string
         registered_office_address:
           $ref: '#/components/schemas/RegisteredOfficeAddress'
+        service_address:
```

Comment:
I _could_ (in theory) change the `RegisteredOfficeAddress` data structure definition to be more generic (e.g. just `Address`) but I didn't do this as this hasn't been done in the partner Java spec file:

https://github.com/companieshouse/private.api.ch.gov.uk-specifications/blob/74a7919a2d8b1ed91d08e23d4dc53707d62a7417/src/main/resources/delta/company-delta-spec.yml#L46

As it is, we're saying that the 'service_address' format is the same as 'registered_office_address', which I think is ok.

Reply:
on line between 45 - 46